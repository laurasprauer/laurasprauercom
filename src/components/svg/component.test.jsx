import React from 'react';
import renderer from 'react-test-renderer';

import SVG from './component';

describe('SVG', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<SVG name="LS-logo" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
