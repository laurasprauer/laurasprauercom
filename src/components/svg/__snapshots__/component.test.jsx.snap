// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SVG renders correctly 1`] = `
<svg
  viewBox="0 0 19 18"
>
  <path
    d="M.248 17V.488H3.56V14h4.176v3H.248ZM13.398 17.192c-.672 0-1.408-.04-2.208-.12-.784-.064-1.48-.16-2.088-.288v-2.688c1.632.128 2.76.192 3.384.192.56 0 .984-.032 1.272-.096.288-.064.504-.176.648-.336a.903.903 0 0 0 .24-.528c.048-.24.072-.576.072-1.008 0-.592-.032-1.008-.096-1.248a.817.817 0 0 0-.408-.552c-.192-.128-.536-.24-1.032-.336l-2.256-.408c-.8-.144-1.352-.584-1.656-1.32-.304-.736-.456-1.8-.456-3.192 0-1.36.192-2.4.576-3.12.4-.72.944-1.208 1.632-1.464s1.56-.384 2.616-.384c.736 0 1.44.048 2.112.144.688.096 1.248.192 1.68.288v2.688a17.758 17.758 0 0 0-2.904-.216c-.576 0-1.024.024-1.344.072-.304.048-.528.136-.672.264a.861.861 0 0 0-.336.552c-.048.24-.072.592-.072 1.056 0 .496.024.856.072 1.08.064.224.168.384.312.48.144.096.392.176.744.24l2.112.384c.56.08 1.032.256 1.416.528.4.256.712.688.936 1.296.224.64.336 1.544.336 2.712 0 1.52-.184 2.664-.552 3.432-.368.768-.88 1.28-1.536 1.536-.64.24-1.488.36-2.544.36Z"
    fill="#fff"
  />
</svg>
`;
